editor/modes/visual-editor/index.js

```diff
@@ -16,7 +16,9 @@ function VisualEditor( { blocks } ) {
 			{ blocks.map( ( uid ) => (
 				<VisualEditorBlock key={ uid } uid={ uid } />
 			) ) }
-			<Inserter />
+			<div className="editor-visual-editor__container">
+				<Inserter />
+			</div>
```

Comment:
>Any context to this change for this PR?

To preserve the common max-width of 700px across blocks while still allowing blocks to escape out of it, the responsibility of assigning the max width was moved from the listing to the block itself. Without the changes here, the inserter would be shown to the far left of the editor canvas since it's not a true block, but should respect aligned with of the blocks (effected with the container).